src/key/key.js

```diff
@@ -150,10 +150,10 @@ class Key {
   toPacketlist() {
     const packetlist = new PacketList();
     packetlist.push(this.keyPacket);
-    packetlist.concat(this.revocationSignatures);
-    packetlist.concat(this.directSignatures);
-    this.users.map(user => packetlist.concat(user.toPacketlist()));
-    this.subKeys.map(subKey => packetlist.concat(subKey.toPacketlist()));
+    packetlist.append(this.revocationSignatures);
```

Comment:
`push` accepts multiple arguments, so you could replace this with
```suggestion
    packetlist.push(...this.revocationSignatures);
```
(and similarly everywhere else). Then you don't need a special method for this